Formula/got-portable.rb

```diff
@@ -0,0 +1,38 @@
+class GotPortable < Formula
+  desc "Distributed version control system"
+  homepage "https://gameoftrees.org"
+  url "https://gameoftrees.org/releases/portable/got-portable-0.74.tar.gz"
+  sha256 "5c495209d161db8adfda0a2c8d2d011be54da8b64d2d8798914cb8b7944876fe"
+  license "ISC"
+  head "https://git.gameoftrees.org/got-portable.git", branch: "linux"
+
+  depends_on "pkg-config" => :build
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "libbsd"
+    depends_on "libmd"
+    depends_on "ossp-uuid"
+  end
+
+  def install
+    if OS.mac?
+      ENV["LIBPANELW_LIBS"] = "-lpanel"
+      ENV["LIBPANELW_CFLAGS"] = " "
```

Comment:
Why do we need these?

Reply:
The MacOS system ncurses doesn't include a `libpanelw`, just `libpanel`, and there's no pkg-config file.  The configure script uses pkg-config to search for panelw, gnupanel, and panel.  If it can't find those, it searches for libpanelw.

Both `LIBPANELW_LIBS` and `LIBPANELW_LIBS` need to be set to prevent the `libpanelw` check from running.  If brew's ncurses is used this isn't necessary.

Reply:
Let's just use Homebrew `ncurses` then.